src/main/kotlin/com/angorasix/projects/management/core/presentation/handler/ProjectsManagementHandler.kt

```diff
@@ -267,20 +269,21 @@ private fun ProjectManagementDto.resolveHypermedia(
     add(getByProjectIdAffordanceLink)
 
     // edit ProjectManagement
-    if (simpleContributor != null) {
-        // @todo check if admin
-        val editProjectManagementRoute = apiConfigs.routes.updateProjectManagement
-        val editProjectManagementLink =
-            Link.of(
-                uriBuilder(request).path(editProjectManagementRoute.resolvePath())
-                    .build().toUriString(),
-            ).withTitle(editProjectManagementRoute.name)
-                .withName(editProjectManagementRoute.name)
-                .withRel(editProjectManagementRoute.name).expand(id)
-        val editProjectManagementAffordanceLink =
-            Affordances.of(editProjectManagementLink).afford(HttpMethod.PUT)
-                .withName(editProjectManagementRoute.name).toLink()
-        add(editProjectManagementAffordanceLink)
+    if (simpleContributor != null && admins != null) {
+        if (admins?.map { it.contributorId }?.contains(simpleContributor.contributorId) == true) {
```

Comment:
Just thinking out loud, and maybe fix in the future across the board as well...do we need `== true`?